core/client/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -164,10 +164,22 @@ public void close() {
       pool.close();
     }
     mBlockWorkerClientPools.clear();
+
+    for (BlockWorkerThriftClientPool pool : mBlockWorkerHeartbeatClientPools.values()) {
```

Comment:
can we use consistent names

`mBlockWorkerHeartbeatClientPools` vs `mFileSystemWorkerClientHeartbeatPools`